identity_storage/src/key_storage/stronghold.rs

```diff
@@ -0,0 +1,365 @@
+// Copyright 2020-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::ed25519;
+use super::JwkGenOutput;
+use super::KeyId;
+use super::KeyStorageError;
+use super::KeyStorageErrorKind;
+use super::KeyStorageResult;
+use super::KeyType;
+use crate::key_storage::JwkStorage;
+use crate::SecretManagerWrapper;
+use async_trait::async_trait;
+use identity_verification::jwk::EdCurve;
+use identity_verification::jwk::Jwk;
+use identity_verification::jwk::JwkParamsOkp;
+use identity_verification::jwk::JwkType;
+use identity_verification::jws::JwsAlgorithm;
+use identity_verification::jwu;
+use iota_stronghold::procedures::Ed25519Sign;
+use iota_stronghold::procedures::GenerateKey;
+use iota_stronghold::procedures::KeyType as ProceduresKeyType;
+use iota_stronghold::procedures::StrongholdProcedure;
+use iota_stronghold::Client;
+use iota_stronghold::ClientError;
+use iota_stronghold::Location;
+use iota_stronghold::Stronghold;
+use rand::distributions::DistString;
+use std::fmt::Display;
+use std::ops::Deref;
+use std::str::FromStr;
+use tokio::sync::MutexGuard;
+
+const ED25519_KEY_TYPE_STR: &str = "Ed25519";
+static IDENTITY_VAULT_PATH: &str = "iota_identity_vault";
+pub(crate) static IDENTITY_CLIENT_PATH: &[u8] = b"iota_identity_client";
+
+/// The Ed25519 key type.
+#[allow(dead_code)]
```

Comment:
If this is correctly exported, the lint should not be necessary, right? Why is this here again?